lib/types/string.js

```diff
@@ -721,6 +728,7 @@ module.exports = Any.extend({
         'string.pattern.invert.name': '{{#label}} with value {:[.]} matches the inverted {{#name}} pattern',
         'string.trim': '{{#label}} must not have leading or trailing whitespace',
         'string.uri': '{{#label}} must be a valid uri',
+        'string.uri.hostname': '{{#label}} is not from given hostname',
```

Comment:
The error might be more useful if it detailed what it expected, like so:
```suggestion
        'string.uri.hostname': '{{#label}} does not match {{#expected}} hostname'
```

Reply:
I made some changes to make the error more useful following the example.